src/api/layer/bankTransactions.ts

```diff
@@ -118,7 +118,23 @@ export const listBankTransactionDocuments = get<{
   errors: unknown
 }>(
   ({ businessId, bankTransactionId }) =>
-    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents`,
+    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents?content_disposition=INLINE`,
+)
+
+export const getBankTransactionDocument = get<{
+  data: S3PresignedUrl
+  errors: unknown
+}>(
+  ({ businessId, bankTransactionId, documentId }) =>
+    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents/${documentId}?content_disposition=INLINE`,
```

Comment:
FYI @tomaszantas  , this does not actually do anything since `inline` is the default content disposition for S3. Instead, I have created an example function that opens an image in a new tab.